.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -83,15 +83,22 @@ Check that the release will work:
 - [ ] Update crate versions, commit the changes to the release branch, and do a release dry-run:
 
 ```sh
-cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad --exclude zebra-scan beta
-# Due to a bug in cargo-release, we need to pass an exact version here
-cargo release version --verbose --execute --allow-branch '*' --package zebra-scan 0.1.0-alpha.1
+# Update everything except for alpha crates and zebrad:
+cargo release version --verbose --execute --allow-branch '*' --workspace --exclude zebrad --exclude zebra-scan --exclude zebra-grpc beta
+# Due to a bug in cargo-release, we need to pass exact versions for alpha crates:
+cargo release version --verbose --execute --allow-branch '*' --package zebra-scan 0.1.0-alpha.4
+cargo release version --verbose --execute --allow-branch '*' --package zebra-grpc 0.1.0-alpha.2
```

Comment:
```suggestion
cargo release version --verbose --execute --allow-branch '*' --package zebra-scan 0.1.0-alpha.3
cargo release version --verbose --execute --allow-branch '*' --package zebra-grpc 0.1.0-alpha.1
```

I think i need to downgrade here in order for the update to be done as part of the release. @upbqdn feel free to make this change here and do the update in the release PR or just leave this and mark as done when doing the release.